desk/app/ord-watcher.hoon

```diff
@@ -1,106 +1,66 @@
-/-  bitcoin, spider, ord, urb, ow=ord-watcher
+/-  bitcoin, spider, ord, urb
 /+  bc=bitcoin, btcio, dbug, default-agent, ol=ord, ul=urb, strandio, verb
 ::
 |%
 +$  card  card:agent:gall
-::
-+$  state-versions
-  $%  state-0:ow
++$  versioned-state  $%(state-0)
++$  state-0
+  $:  %0
+      rpc=req-to:btcio
+      ord-state=state:ord
   ==
-::
 --
+::
 %-  agent:dbug
 ^-  agent:gall
-=|  state-0:ow
+=|  state-0
 =*  state  -
 %+  verb  &
 =<
 |_  =bowl:gall
 +*  this   .
     def    ~(. (default-agent this %|) bowl)
-    cor    ~(. +> bowl)
 ::
+::  Tell Jael to subscribe to us for PKI updates,
+::  initialize an ord-core, start a thread to
+::  fetch and process the first batch of blocks,
+::  and start a timer to fetch again.
 ++  on-init
-  :-  :~  [%pass /lo %arvo %j [%listen *(set ship) [%| dap.bowl]]]
+  ^-  (quip card _this)
+  =/  new-rpc  ['http://localhost:18443' [%basic 'bitcoinrpc:bitcoinrpc']]
+  =/  new-ord-state  [[start-hash:urb start-height:urb] ~ ~ ~]
```

Comment:
Initializing the entire ord-state as one thing seems fine; instead of `~`s I'd rather explicitly bunt the various maps.

```hoon
=/  init-ord-state
  $:  [start-hash:urb start-height:urb]
      *sont-map
      *insc-ids
      *unv-ids
  ==
```